pyuvsim/tests/test_simsetup.py

```diff
@@ -223,6 +223,25 @@ def test_param_reader():
         yield (check_param_reader, n)
 
 
+def test_tele_parser():
+    """
+    Check a variety of cases not already tested by param reader
+    """
+    # check no tele config passed
+    tdict = dict(array_layout=os.path.join(SIM_DATA_PATH, 'test_layout_6ant.csv'))
+    nt.assert_raises(KeyError, pyuvsim.simsetup.parse_telescope_params, tdict)
+    tdict['telescope_location'] = '(-30.72152777777791, 21.428305555555557, 1073.0000000093132)'
+    nt.assert_raises(KeyError, pyuvsim.simsetup.parse_telescope_params, tdict)
```

Comment:
Again maybe nit-picky but moving forward it would be nice to make sure the correct errors are raised instead of just the correct type of error. These can easily be implemented in a `with` statement and checking the text of the error.

Reply:
+1

Reply:
There are many `assert_raises` statements throughout the tests... would it make sense to make this change a part of the nose to pytest conversion? It seems pointless to implement it now if we're just going to replace it with pytest later.

Reply:
The statements are almost identical in `nose` and `pytest` honestly.

something like this example from an older build of pyuvdata:
```
    with nt.assert_raises(ValueError) as cm:
        uv2.reorder_pols(order=[3, 2, 1])
    ex = cm.exception  # raised exception is available through exception property of context
    nt.assert_true(ex.args[0].startswith('If order is an index array, it must'))
```
and the permalink
https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/63b61f84f354448ae500c6e8d6e50c3329dedcc4/pyuvdata/tests/test_uvdata.py#L1157-L1160

it's very easy to convert this to nose.

Reply:
Okay! I'll make the changes.